meta/releasing.md

```diff
@@ -9,45 +9,44 @@ machinery works.
+* Q1: January 20-27 (if needed before FOSDEM) or February 21-28.
+* Q2: May 15-22.
+* Q3: August 1-7 or August 20-27 depending on feature requirements.
 * Q4: November 1-15 (before recurring November conflicts, like IETF).
 
-The SCT aims to have dates picked out by:
-
-* Q1: January 10.
-* Q2: May 1.
-* Q3: August 1.
-* Q4: October 15.
+The SCT aims to have dates picked out 2 weeks before the chosen release date. When
+possible, releases should be scheduled for Thursdays and Fridays to allow a few
+consecutive business days for identifying blockers.
 
 When a release date is picked, a [checklist](https://github.com/matrix-org/matrix-spec/issues/new?assignees=&labels=release-blocker&projects=&template=release.md&title=Matrix+1.X)
-issue is created to track details of the release. Release blockers should continue to
-be accepted up until 7 calendar days prior to the release date.
+issue is created to track details of the release. Release blockers should continue
+to be accepted at the discretion of whoever is doing the release (typically, blockers
+should be allowed up to 1-2 days before the release date).
 
 **Release dates are not promises.** The SCT reserves the ability to change, cancel,
 postpone, etc a release for any reason. Do not rely on a release happening on a given
 day until the release has actually happened & blog post published.
 
-Once a release is scheduled, the SCT will begin planning what the next release is
+Once a release is *scheduled*, the SCT will begin planning what the next release is
 expected to look like. The plan should be included in the spec release blog post,
 and be ready for execution on spec release day. Plans are guides and not promises.
 
-A blog post for the SCT members to review should be ready at minimum 1 week before
-the target release date. 1-2 days before the release itself, the prerequisite steps
-below are executed to ensure the spec release can go ahead.
+A blog post for the SCT members to review should be ready 2-3 days prior to the
+release at minimum. Preferably a week in advance.
+
+1-2 days before the release itself, the prerequisite steps below are executed to
+ensure the spec release can go ahead.
 
 ## Release composition
 
 *This section is a work in progress.*
 
-Mentioned above, the SCT aims to have spec releases quarterly. Each quarter has a
-slightly different theme to it:
+Spec releases do not currently have attached themes, though when planning a release
+a broad theme may be considered. Ideally, each release contains a "hero feature"
+which is highlighted in the later blog post.
 
-* Q1: Massive feature release, if possible. This generally happens thanks to FOSDEM.
-* Q2: Regular feature release, if possible.
-* Q3: Momentum-continuing feature release, if possible.
-* Q4: Preferably a maintenance release, but will accept features per normal.
+Maintenance-only releases are discouraged, but typical in Q4 to give implementations
+a small bit of reprieve from feature development.
```

Comment:
Are they discouraged? I mean, sure, a headline feature is nice for marketing and publicity, but *discouraging* a release without such a feature seems a bit harsh.

Reply:
Discouraged might be the wrong word here. This is trying to say there should always be a plan of some sort to avoid organic releases. Maybe we discourage organic releases?

Reply:
I'm not entirely sure what you mean by an organic release, but what's so bad about them?

Reply:
Organic releases are the ones where we didn't really have a plan and just cut the spec on schedule with whatever random changes happened to be there. Preferably, we put a bit of thought and planning into the spec's progression instead of letting it coast through releases with just clarification changes.

Reply:
> Preferably, we put a bit of thought and planning into the spec's progression instead of letting it coast through releases with just clarification changes.

I mean sure, planning for the spec's progression is a good thing, but if, for whatever reason, we end up with a release of mostly maintenance stuff, that seems fine?

I just don't really feel like this belongs in the release process.

But not a strong opinion.

Reply:
paragraph dropped